Bugzilla/DB.pm

```diff
@@ -111,8 +111,11 @@ sub connect_shadow {
     my $connect_params = dclone(Bugzilla->localconfig);
     $connect_params->{db_host} = Bugzilla->get_param_with_override('shadowdbhost');
     $connect_params->{db_name} = Bugzilla->get_param_with_override('shadowdb');
-    $connect_params->{db_port} = Bugzilla->get_param_with_override('shadowport');
-    $connect_params->{db_sock} = Bugzilla->get_param_with_override('shadowsock');
+    $connect_params->{db_port} = Bugzilla->get_param_with_override('shadowdbport');
+    $connect_params->{db_sock} = Bugzilla->get_param_with_override('shadowdbsock');
+    $connect_params->{db_user} = Bugzilla->get_param_with_override('shadowdbuser');
```

Comment:
Sorry I didn't mention in the bug -- don't use the get_param_with_override stuff.
That exists because the original shadow_ params exist in the data/params file, but cloudops can only modify the localconfig file.

Just use `Bugzilla->localconfig->{KEY}` for the new entries.